src/main/java/com/yahoo/bullet/querying/evaluators/NAryOperations.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright 2019, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.querying.evaluators;
+
+import com.yahoo.bullet.typesystem.Type;
+import com.yahoo.bullet.typesystem.TypedObject;
+
+import static com.yahoo.bullet.querying.evaluators.Evaluator.NAryOperator;
+
+public class NAryOperations {
```

Comment:
These operations classes should probably go in an operations package. Either querying.evaluators.operations or querying.operations?